cluster-spec.json

```diff
@@ -0,0 +1,6 @@
+{
+  "spark_version": "7.3.x-scala2.12",
+  "num_workers": 1,
+  "node_type_id": "i3.xlarge"
```

Comment:
Is there anywhere to specify the lifecycle of the instance as "spot"? This would save $

Reply:
Looks like doable, see https://docs.databricks.com/clusters/cluster-config-best-practices.html#on-demand-and-spot-instances. We can ask them when showing this example.